frontend/modules/recipe/components/Ratings.js

```diff
@@ -4,19 +4,19 @@ require("./../css/ratings.scss");
 
 export default React.createClass({
   render: function() {
-    var tmp = '......';
-    var start_map = tmp.split(".", this.props.stars);
-    var empty_star_map = tmp.split(".", 5 - this.props.stars);
-    var stars = start_map.map(function(name, index){
+    const stars = this.props.stars ? this.props.stars : 0;
```

Comment:
We can address the bigger problem or users entering large numbers in https://github.com/RyanNoelk/OpenEats/issues/79